src/components/Options/index.tsx

```diff
@@ -107,25 +112,27 @@ export default class Options extends Component<Props, State> {
 
     return (
       <div class={`${style.options}${className ? (' ' + className) : ''}`}>
-        <p>Quantization</p>
-        <label>
-          <input
-            name="quantizer.enable"
-            type="checkbox"
-            checked={!!preprocessorState.quantizer.enabled}
-            onChange={this.onPreprocessorEnabledChange}
-          />
-          Enable
-        </label>
-        {preprocessorState.quantizer.enabled ? (
-          <QuantizerOptionsComponent
-            options={preprocessorState.quantizer}
-            onChange={this.onQuantizerOptionsChange}
-          />
-        ) : (
-          <div/>
+        {encoderState.type !== 'identity' && (
+          <div>
+            <p>Quantization</p>
+            <label>
+              <input
+                name="quantizer.enable"
```

Comment:
`enabled`?

Reply:
I only changed the indenting as part of this PR, but happy to rename it.